BankingKataSpecs/AccountSpecs/Features/DepositCash.feature

```diff
@@ -0,0 +1,10 @@
+﻿Feature: DepositCash
+	In order to deposit cash into my account
+	As a foo user
+	I want to be able to deposit Cash
+
+@mytag
+Scenario: Depositing my cash
+	Given I create an account with 0 cash
```

Comment:
Is there any good Resharper or other (support) for tests like these? e.g. if I had misspelled one of these conditions and want to correct it, to I have to do a find/replace on the string or can I do an automated refactoring?